<launch>

  <node name="key_teleop" pkg="jog_teleop" type="key_teleop" output="screen" />

  <node name="jog_arm_server" pkg="jog_arm" type="jog_arm_server" output="screen" >
    <param name="parameter_ns" type="string" value="jog_arm_server" />
    <rosparam command="load" file="$(find jog_arm)/config/jog_settings.yaml" />
  </node>

</launch>
